Makefile

```
@@ -4,9 +4,9 @@ all: check
 .PHONY: check
 check:
 	cargo +nightly fmt --check
-	cargo audit
-	cargo clippy
-	cargo test
+	cargo audit --deny warnings
+	cargo clippy --all-targets -- -D warnings
+	cargo test --all-targets
```

I don't know what that does; I think the tests _are_ a target?

```suggestion
	cargo test
```